katas/langtonant/alfredo/main.js

```diff
@@ -0,0 +1,20 @@
+import { class1 } from './module1.js';
```

Comment:
Hi Alfredo, lets try to improve the names for the modules you have, something that could be more descriptive and can self-explain the responsabilities for each piece of code you have